tests/ballerina-unit-test/src/test/resources/test-src/expressions/conversion/native-conversion-negative.bal

```diff
@@ -30,13 +30,16 @@ function testStructToStruct () returns (Student) {
 }
 
 type Info record {
-    byte[] infoBlob;
+    byte[] infoBlob = [];
     !...
 };
 
-function testStructWithIncompatibleTypeToJson () returns (json) {
+function testStructWithIncompatibleTypeToJson () returns json|error {
```

Comment:
Do we need |error part here, since we are not returning an error or not using `check` anymore.